research-app/src/App.vue

```diff
@@ -624,6 +652,12 @@ export default class App extends WWTAwareComponent {
       screenfull.on('change', this.onFullscreenEvent);
     }
 
+    (async() => {
+      this.loadImageCollection({ url: this.hipsUrl, loadChildFolders: true });
+    })();
+
+    
+
```

Comment:
Can this just be a call to `loadImageCollection` without the async wrapper? I don't get any compiler errors if I do that ...

Also, I like to be very generous with my vertical spacing, but I generally keep spaces to one blank line at most within a function. (These comments are indicating that we should look into setting up automatic code formatting once we're past this deadline — this isn't stuff that needs to be managed manually.)